src/server/Listener.js

```diff
@@ -43,20 +43,25 @@ class Listener {
 	add (options) {
 		const {uri, method} = options;
 		this.mocks[uri] = this.mocks[uri] || {};
-
-		// Register mock if it hasn't been registered before
-		if (!this.mocks[uri][method]) {
-			this.app[method.toLowerCase()](uri, (req, res) => {
-				return this.mocks[uri][method].handler(req, res);
-			});
+		const oldMock = this.mocks[uri][method];
```

Comment:
Isn't `currentMock` one being added? See the last if clause 
```
if (!oldMock) {
			this.app[method.toLowerCase()](uri, (req, res) => {
				return this.mocks[uri][method].handler(req, res);
			});
		}
```
does not read that good if it's called `currentMock`, because it has been already overridden